docs/iot/iot-partners/anchor.md

```diff
@@ -0,0 +1,72 @@
+# Tizen IoT with Anchor SoM Module
+
+> The Fastest Way to design your IoT Applications
+
+## Anchor5
+
+**Anchor5** is a system-on-module(SoM) built in a 32-bit RISC application processor that is based on an quad-core ARM® quad Cortex®-A9 cores architecture with max 1.4GHz speed, containing DRAM provided 6.4 GB/s memory bandwidth with flash memory, camera and display interfaces displayed Full HD, a full complement of digital I/O and analog inputs, and world class wireless connectivity with IEEE802.11a/b/g/n/ac and Bluetooth(Classic+BLE) inside a ultra compact package that is just 38x45mm.
```

Comment:
@johnny1411 @mijin85cho Content is still plagiarized from the Specification sheet of Anchor5 SoM. As suggested in the previous comment https://github.com/Samsung/tizen-docs/pull/850/files#r331434816 can we check for copyright permissions from Anchor SoM officials.

Reply:
The committer(johnny1411) is from Dignsys.

Reply:
Spec. sheet file was moved to Anchor Official web site